zeth_contracts/.solhint.json

```diff
@@ -0,0 +1,16 @@
+{
+    "extends": "solhint:recommended",
+    "plugins": [],
+    "rules": {
+        "var-name-mixedcase": "off",
+        "func-name-mixedcase": "off",
+        "contract-name-camelcase": "off",
+        "no-empty-blocks": "off",
+        "compiler-version": "off",
+        "reason-string": "off",
+        "avoid-suicide": "error",
+        "no-inline-assembly": "off",
+        "avoid-sha3": "warn",
+        "func-visibility": "error"
+    }
```

Comment:
Just FYI, I tried adding `"max-line-length": ["error",80]`, but this catches comments (e.g. long URLs).  Would be nice to have that at some point, though, perhaps using `// solhint-diable-next-line ...` where long lines are unavoidable.

Reply:
Thanks #334 was merged in this branch.